### ISSUE TITLE
Invalid focus after opening select menu when screen reader enabled

### ISSUE THREAD

**Issue by KarolDawidziuk** [ckeditor/ckeditor4#5201]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Enable the screen reader: NVDA or JAWS.
2. Open the http://192.168.0.2:1030/tests/plugins/menubutton/manual/menubuttonscreenreader#child test
3. Put caret into the editable.
4. Move your focus to the toolbar by pressing `alt+f10`.
5. Open the select list with the `space` key.

### Expected result

Focus is moved to the first element in list.

### Actual result for NVDA.

https://user-images.githubusercontent.com/18471998/167867601-6dacc888-06ef-4598-b08b-e3bc757d372a.mov

Focus is not visible.

### Actual result for JAWS.

Focus is jumped to the second position on the list.

https://user-images.githubusercontent.com/18471998/167866403-943428b5-dee9-4294-a9d3-a62c36ec7236.mov

## Other details

I didn't notice this issue with `VoiceOver` on macOS.

* Browser: IE, Chrome
* OS: Windows
* CKEditor version: 4.19.0 (pre-released)

**Comment by sculpt0r** [comment]
I didn't encounter such behaviour with JAWS 2021 & IE 8 (Win7)

**Comment by KarolDawidziuk** [comment]
Yes, it only happens the first time. I forgot to mention it in the description. Already updated.

**Comment by Comandeer** [comment]
I can confirm the issue with JAWS in IE11. However, it seems to be visible only in the manual test, I could not reproduce it on the sample with other plugins.
